### PR TITLE
[README] Translate the half of README into Chinese & Correct some mistakes in the original README

### PR DESCRIPTION
**Pull request by AC-Phoenix**
1. Translate the half of README into Chinese.
2. Correct some mistakes in the original README.

### CODE REVIEW

File: README.zh-CN.md [hunk 1]
```diff
@@ -18,92 +18,92 @@
...
-- Or: Onboard your test device to an existing network with a minimized effort.
+- 搭建：创建一个新的云测试网络。
+- 加入：以最小的代价将你的测试设备部署到现有的网络上。
 
-Capabilities of Hydra Lab include:
-- Scalable test device management under the center-agent distributed design; Test task management and test result visualization.
-- Powering [Android Espresso Test](https://developer.android.com/training/testing/espresso)
-- Appium(Java) test on different platforms: Windows/iOS/Android/Browser/Cross-platform
-- Case-free test automation: Monkey test, Smart exploratory test
+Hydra Lab的功能包括：
```
Comment (zhou9584) [inline README.zh-CN.md:27]: 功能-> 特性 or 特色？

File: README.zh-CN.md [hunk 2]
```diff
@@ -18,92 +18,92 @@
...
 ```
 
-**Step 2: build and run Hydra Lab agent service.**
+**第2步：构建并运行Hydra Lab agent服务。**
 
 ```bash
-# In project root, copy the sample config file and update the:
-# YOUR_AGENT_NAME, YOUR_REGISTERED_AGENT_ID, and YOUR_REGISTERED_AGENT_SECRET.
+# 在项目根目录下，复制示例配置文件并更新：
+# YOUR_AGENT_NAME, YOUR_REGISTERED_AGENT_ID, 和 YOUR_REGISTERED_AGENT_SECRET.
```
Comment (zhou9584) [inline README.zh-CN.md:99]: 多了一个逗号

File: README.zh-CN.md [hunk 3]
```diff
@@ -18,92 +18,92 @@
...
+> 构建和运行过程需要用到JDK | NPM | Android SDK平台工具。
 
-**Step 1: build and run Hydra Lab center service.**
+**第1步：构建并运行Hydra Lab center服务。**
 
 ```bash
-# In project root, switch to react folder to build the Web front.
+# 在项目根目录，切换到react文件夹来构建Web前端文件。
 cd react
 npm ci
```
Comment (zhou9584) [inline README.zh-CN.md:84]: 上次在wiki加的备注，也许也可以在这里加一下，还有readme.md

File: README.zh-CN.md (hunk 2 above)
Comment (AC-Phoenix) [reply README.zh-CN.md]: Sharp Eyes! Fixed.

File: README.zh-CN.md [hunk 4]
```diff
@@ -18,92 +18,92 @@
...
-Capabilities of Hydra Lab include:
-- Scalable test device management under the center-agent distributed design; Test task management and test result visualization.
-- Powering [Android Espresso Test](https://developer.android.com/training/testing/espresso)
-- Appium(Java) test on different platforms: Windows/iOS/Android/Browser/Cross-platform
-- Case-free test automation: Monkey test, Smart exploratory test
+Hydra Lab的功能包括：
+- center-agent分布式设计下的可扩展测试设备管理；测试任务管理和测试结果可视化。
+- 支持 [Android Espresso Test](https://developer.android.com/training/testing/espresso)。
+- 支持在不同平台上进行Appium(Java)测试：Windows/iOS/Android/浏览器/跨平台。
+- 无用例的自动化测试：猴子测试，智能探索测试
```
Comment (RaccoonLi) [inline README.zh-CN.md:31]: Monkey test 不翻译会不会好一点，中文译名挺奇怪的 https://baike.baidu.com/item/Monkeytest/836393
另外 Android 官方也没有翻译
image

File: README.zh-CN.md (hunk 3 above)
Comment (AC-Phoenix) [reply README.zh-CN.md:84]: Done.

File: README.zh-CN.md (hunk 1 above)
Comment (AC-Phoenix) [reply README.zh-CN.md]: Done.

File: README.zh-CN.md (hunk 4 above)
Comment (AC-Phoenix) [reply README.zh-CN.md]: Done. Thanks!

File: README.zh-CN.md [hunk 5]
```diff
@@ -18,92 +18,92 @@
...
 
-Capabilities of Hydra Lab include:
-- Scalable test device management under the center-agent distributed design; Test task management and test result visualization.
-- Powering [Android Espresso Test](https://developer.android.com/training/testing/espresso)
-- Appium(Java) test on different platforms: Windows/iOS/Android/Browser/Cross-platform
-- Case-free test automation: Monkey test, Smart exploratory test
+Hydra Lab的功能包括：
+- center-agent分布式设计下的可扩展测试设备管理；测试任务管理和测试结果可视化。
+- 支持 [Android Espresso Test](https://developer.android.com/training/testing/espresso)。
+- 支持在不同平台上进行Appium(Java)测试：Windows/iOS/Android/浏览器/跨平台。
```
[the remark this answers is not in the document]
Comment (AC-Phoenix) [reply README.zh-CN.md]: Done. thanks!

File: README.zh-CN.md [hunk 6]
```diff
@@ -18,92 +18,94 @@
 <span id="what-is"></span>
 ## Hydra Lab是什么？
```
Comment (olivershen-wow) [inline README.zh-CN.md:19]: Maybe unify the usage of blank between Chinese words and English phrases?
Currently usage involves some blanks, but not for every single place.

File: README.zh-CN.md [hunk 7]
```diff
@@ -18,92 +18,94 @@
...
-|Web (Browser)| &#10004;     | x | 
+|Web (浏览器)| &#10004;     | x | 
 
 <span id="quick-start"></span>
-### Quick guide on out-of-box Uber experience
+### 开箱即用的Uber快速指南
 
-Hydra Lab uses [Azure Blob Storage](https://azure.microsoft.com/en-us/products/storage/blobs/) as cloud file storage solution to persist log files, video, app package, etc. Please go to your Azure portal, open an Azure blob storage account, get the [connection string](https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string),
-and place it in the environment variable with the name of BLOB_CONNECTION_STR.
+Hydra Lab使用[Azure Blob Storage](https://azure.microsoft.com/en-us/products/storage/blobs/)作为云文件存储解决方案，以持久化存储日志文件、视频、应用包等。请访问你的Azure门户，打开一个Azure Blob存储账户，获取[connection string](https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string)。
```
Comment (olivershen-wow) [inline README.zh-CN.md:54]: Storage => 存储

File: README.zh-CN.md [hunk 8]
```diff
@@ -18,92 +18,94 @@
...
 > Url: http://localhost:9886/portal/index.html#/ (or your custom port).
 
-Enjoy starting your journey of exploration!
+开始享受你的探索之旅吧!
 
-**Note: out-of-box Uber now only provides the Espresso test feature for Android, please refer to this section for more features: [For Hydra Lab User](#for-user)** 
+**注意：开箱即用的Uber现在只为安卓系统提供Espresso测试功能，更多的功能请参考本节：[对于Hydra Lab用户](#for-user)** 
 
-### Quick guide on build and run
+### 构立和运行的快速指南
```
Comment (olivershen-wow) [inline README.zh-CN.md:73]: 构建

File: README.zh-CN.md [hunk 9]
```diff
@@ -18,92 +18,94 @@
...
 > docker pull ghcr.io/microsoft/hydra-lab-uber:latest
 
-**Step 2. run on your machine with BLOB_CONNECTION_STR**
+**第2步. 在你的机器上运行，并使用BLOB_CONNECTION_STR作为参数**
 > docker run [-p 9886:9886] [--name=hydra-lab] -e BLOB_CONNECTION_STR=${BLOB_CONNECTION_STR} ghcr.io/microsoft/hydra-lab-uber:latest
 
-**Step 3: visit front-end page and view your connected devices**
+**第3步. 访问前端页面并查看你的已连接设备**
 
 > Url: http://localhost:9886/portal/index.html#/ (or your custom port).
```
Comment (olivershen-wow) [inline README.zh-CN.md:67]: 或自定义的端口号

File: README.zh-CN.md (hunk 6 above)
Comment (AC-Phoenix) [reply README.zh-CN.md]: Done, thanks.

File: README.zh-CN.md (hunk 7 above)
Comment (AC-Phoenix) [reply README.zh-CN.md]: Done.

File: README.md [hunk 10]
```diff
@@ -75,7 +75,7 @@ Or simply run with the env parameter -e:
 docker run [-p 9886:9886] [--name=hydra-lab] -e BLOB_CONNECTION_STR=${YOUR_BLOB_CONNECTION_STR} ghcr.io/microsoft/hydra-lab-uber:latest
 ```
 
-**Step 3: visit front-end page and view your connected devices**
+**Step 3. visit front-end page and view your connected devices**
```
[the remark this answers is not in the document]
Comment (AC-Phoenix) [reply README.md:78]: Done.

File: README.zh-CN.md (hunk 8 above)
Comment (AC-Phoenix) [reply README.zh-CN.md]: Fixed.

File: README.md [hunk 11]
```diff
@@ -28,9 +28,9 @@ It enables dev team to quickly build a self-manageable and intelligent cloud tes
```
[the remark this answers is not in the document]
Comment (AC-Phoenix) [reply README.md:28]: Done. thanks!

File: README.zh-CN.md (hunk 9 above)
Comment (AC-Phoenix) [reply README.zh-CN.md]: Done. thanka.

**Review by zhou9584 — APPROVED** [review]

**Pull request merged**